test/CadlRanchProjectsNonAzure/authentication/http/custom/src/Generated/CustomClient.cs

```diff
@@ -96,7 +96,7 @@ public virtual ClientResult Valid(RequestOptions options = null)
         /// <list type="bullet">
         /// <item>
         /// <description>
-        /// This <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md">protocol method</see> allows explicit creation of the request and processing of the response for advanced scenarios.
+        /// This <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/System.ClientModel/samples/ServiceMethods.md#protocol-methods">protocol method</see> allows explicit creation of the request and processing of the response for advanced scenarios.
```

Comment:
Non-blocking question: do we want the string "Azure" to appear here?  We could use an aka.ms link instead of the full path if we wanted to remove Azure from unbranded client code and docs.

Reply:
I would change this to [aka.ms/azsdk/net/protocol-methods](https://aka.ms/azsdk/net/protocol-methods) as you suggested in https://github.com/annelo-msft/openai-in-typespec/pull/1#discussion_r1550115744